packages/longhorn/longhorn.patch

```diff
@@ -0,0 +1,774 @@
+     type: string
+     label: Longhorn CSI Attacher Image Tag
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.provisionerImage
++  - variable: image.csi.provisioner.repository
+     default: longhornio/csi-provisioner
+-    description: "Specify CSI provisioner image. Leave blank to autodetect."
++    description: "Specify CSI provisioner image repository. Leave blank to autodetect."
+     type: string
+-    label: Longhorn CSI Provisioner Image
++    label: Longhorn CSI Provisioner Image Repository
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.provisionerImageTag
++  - variable: image.csi.provisioner.tag
+     default: v1.4.0
+     description: "Specify CSI provisioner image tag. Leave blank to autodetect."
+     type: string
+     label: Longhorn CSI Provisioner Image Tag
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.nodeDriverRegistrarImage
++  - variable: image.csi.nodeDriverRegistrar.repository
+     default: longhornio/csi-node-driver-registrar
+-    description: "Specify CSI Node Driver Registrar image. Leave blank to autodetect."
++    description: "Specify CSI Node Driver Registrar image repository. Leave blank to autodetect."
+     type: string
+-    label: Longhorn CSI Node Driver Registrar Image
++    label: Longhorn CSI Node Driver Registrar Image Repository
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.nodeDriverRegistrarImageTag
++  - variable: image.csi.nodeDriverRegistrar.tag
+     default: v1.2.0
+     description: "Specify CSI Node Driver Registrar image tag. Leave blank to autodetect."
+     type: string
+     label: Longhorn CSI Node Driver Registrar Image Tag
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.resizerImage
++  - variable: image.csi.resizer.repository
+     default: longhornio/csi-resizer
+-    description: "Specify CSI Driver Resizer image. Leave blank to autodetect."
++    description: "Specify CSI Driver Resizer image repository. Leave blank to autodetect."
+     type: string
+-    label: Longhorn CSI Driver Resizer Image
++    label: Longhorn CSI Driver Resizer Image Repository
+     group: "Longhorn CSI Driver Images"
+-  - variable: csi.resizerImageTag
++  - variable: image.csi.resizer.tag
+     default: v0.3.0
+     description: "Specify CSI Driver Resizer image tag. Leave blank to autodetect."
+     type: string
+diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/templates/_helpers.tpl packages/longhorn/charts/templates/_helpers.tpl
+--- packages/longhorn/charts-original/templates/_helpers.tpl
++++ packages/longhorn/charts/templates/_helpers.tpl
+@@ -36,3 +36,12 @@
+ app.kubernetes.io/instance: {{ .Release.Name }}
+ app.kubernetes.io/version: {{ .Chart.AppVersion }}
+ {{- end -}}
++
++
++{{- define "system_default_registry" -}}
++{{- if .Values.global.systemDefaultRegistry -}}
++{{- printf "%s/" .Values.global.systemDefaultRegistry -}}
```

Comment:
This changed to be global.cattle, I updated the doc with better links: https://github.com/rancher/charts/blob/b55e6ec01907964723942fdbd9f63b9afef7a1ed/packages/rancher-cis-benchmark/charts/templates/_helpers.tpl#L6-L12

Need to update values.yaml also